iothub/service/src/ApiResources.resx

```diff
@@ -120,9 +120,6 @@
   <data name="ConnectionStringIsNotWellFormed" xml:space="preserve">
     <value>The connection string is not well formed.</value>
   </data>
-  <data name="HostNameIsNull" xml:space="preserve">
```

Comment:
Removed one case where we spot checked was not used. We should go through this file and remove all the others that aren't referenced.